### ISSUE TITLE
9.5.0 version does not complete startup on Fedora 33

### ISSUE THREAD

**Issue by venkatcc** [a85/POSTMan-Chrome-Extension#10507]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Postman fails to start. "Preparing your workspaces.... This might take a few minutes" is all I see.

```
$ ./Postman                                                                                                                                                                                                                            
The disableGPU setting is set to undefined
Not disabling GPU
Main~createDefaultWorkingDir - Default working dir creation already attempted
1639759131301 main info "Booting Postman 9.5.0, linux-5.14.18-100.fc33.x86_64 on x64"
1639759131302 main info "EventBus~initialize - Success"
1639759131304 main info "Proxy configuration has not been setup"
1639759131305 main info "CloudProxyManager~init - Success"
1639759131309 main info "UpdateHandler~init - Success"
1639759131503 main info "RuntimeIPCAgent~started: Success"
1639759131504 main info "LinuxAutoUpdater~Cleanup - Initial cleanup successful"
1639759131505 main info "LeaderSelection: Initialized successfully"
1639759131639 main info "GPU detected VID 4318 DID 7425 ACTIVE true"
1639759131761 main info "window-manager~newSharedWindow: Shell loaded"
1639759132982 main info "Bootstrap-models~bootstrap - Success"
```


### Steps To Reproduce

Start Postman and wait for it to boot completely.

### Screenshots or Videos

_No response_

### Environment Information

```markdown
- Operating System: Fedora 33
- Platform Type: Native App
- Postman Version: 9.5.0
```


### Additional Context?

Updated from an earlier version and I got stuck. Then deleted and reinstalled, but still have the same problem.

**Comment by vstrelstov** [comment]
Got the same issue on Manjaro, no matter if it's AUR or Snap version

**Comment by postman-staff** [comment]
@venkatcc @vstrelstov Sorry to hear about this experience.

If possible, would you [please submit a ticket from here](https://support.postman.com/hc/en-us/requests/new/) and share your app's logs to help us debug the issue next? [This article](https://support.postman.com/hc/en-us/articles/360047765713-Where-can-I-find-logs-for-the-Postman-Application-) should help you further on the same.

* * *
Also do try renaming (or _clearing_) the local files for your Postman installation generally resolves this issue. You can find full instructions for doing so in the below article.


- https://support.postman.com/hc/en-us/articles/4410785125271

Let us know how it goes and we will take it from there.

via @Raj-postman | Postman Support

**Comment by venkatcc** [comment]
@Raj-postman Sorry, I'm not sure what you mean by "submit a ticket from
here". What is "here"?

That said, I cleared local files using the procedure outlined in the linked
support article in your previous email. I still see the same behavior.

This is what I see when I try to start the application from the command
line (after clearing local files):
```
$ ./Postman

            The disableGPU setting is set to undefined
Not disabling GPU
Main~createDefaultWorkingDir - Default working dir creation already
attempted
1639986188373 main info "Booting Postman 9.5.0,
linux-5.14.18-100.fc33.x86_64 on x64"
1639986188376 main info "EventBus~initialize - Success"
1639986188378 main info "Proxy configuration has not been setup"
1639986188378 main info "CloudProxyManager~init - Success"
1639986188383 main info "UpdateHandler~init - Success"
1639986188577 main info "RuntimeIPCAgent~started: Success"
1639986188578 main info "LinuxAutoUpdater~Cleanup - Initial cleanup
successful"
1639986188580 main info "LeaderSelection: Initialized successfully"
1639986188973 main info "GPU detected VID 4318 DID 7425 ACTIVE true"
1639986189117 main info "window-manager~newSharedWindow: Shell loaded"
1639986189141 main info "Bootstrap-models~bootstrap - Success"
```

Additional logs from "renderer-shell.log":

```
[307171][1639987022239][shell][info]["EventBus~initialize - Success"]
[307171][1639987022244][shell][info]["bootShellCrashReporter~ Success"]
[307171][1639987022251][shell][info]["shellController~init: Success"]
[307171][1639987022251][shell][info]["shellMeta~init: Success"]
[307171][1639987022252][shell][info]["partitionMigrationService: State of V7 partition:"]
[307171][1639987022252][shell][info]["partitionMigrationService: V7 partitions",[]]
[307171][1639987022252][shell][info]["partitionMigrationService: V7 lastActiveUser",null]
[307171][1639987022252][shell][info]["partitionMigrationService: V7 activePartition",null]
[307171][1639987022252][shell][info]["partitionMigrationService: V7 isPartitionDirty",null]
[307171][1639987022252][shell][info]["partitionMigrationService~init: Success"]
[307171][1639987022253][shell][info]["partitionMigrationService~runV8PartitionsIntegrity: Two way migration~ removing the following partitions from v8",[]]
[307171][1639987022253][shell][info]["userPartitionService~_rectifyTheActivePartitionOfApplication: Setting the active partition to ",null]
[307171][1639987022253][shell][info]["userPartitionService~init: Success"]
[307171][1639987022254][shell][info]["userPartitionService: State of v8 partition system:"]
[307171][1639987022254][shell][info]["userPartitionService: List of partitions:",[{"partitionId":"feaeff1b-caab-403f-bc9d-cf812821b32b","namespace":"scratchPad","userId":0,"teamId":0,"isDirty":false}]]
[307171][1639987022254][shell][info]["userPartitionService: users global meta:",{"activePartition":null}]
[307171][1639987022254][shell][info]["shell~init~loadView: loading scratchpad partition"]
```

**Comment by venkatcc** [comment]
Ah, I see what you meant by "submit a ticket from here". The hyperlink was not a link in the email message, but is only visible on the github page. 

Thanks!

**Comment by postman-staff** [comment]
@venkatcc Sorry for that, the link for submitting a support request is: https://support.postman.com/hc/en-us/requests/new/

This is to make sure all of the details shared are communicated via the internal channel and they remain safe :)

* * *
Could you check the below things to see if that helps by any chance:


- Disable GPU as per this article here: https://github.com/postmanlabs/postman-app-support/issues/4594#issuecomment-391601621
   or do run the below command in your command line:

_> setx POSTMAN_DISABLE_GPU "true" /m_

- Install OpenSSL if not already present, preferably v.1.x based on your OS-bit from here.
- You may try to enable/ disable hardware acceleration to see if that helps.

![image](https://user-images.githubusercontent.com/47771757/146886132-3930ca24-def9-40f3-8e7a-a98386120ae2.png)

[More details here](https://wccftech.com/how-to/how-to-enable-hardware-accelerated-gpu-scheduling-on-windows-11/) and [Reference video.](https://www.youtube.com/watch?v=xpINgLCnosM)

* * *
Once done - Remove your Postman’s local data:
  > https://support.postman.com/hc/en-us/articles/4410785125271

via @Raj-postman | Postman Support

**Comment by vstrelstov** [comment]
I've created a ticket. Deleting local data has no effect, Postman still refuses to start

**Comment by vstrelstov** [comment]
I've created the [request:](https://support.postman.com/hc/en-us/requests/93791)
Deleting Postman's local data had no effect, but nevertheless, I managed to start it with the key --disable-gpu. But the strange thing is the fact that Postman seems to be running _with_ hardware acceleration ignoring the key
![изображение](https://user-images.githubusercontent.com/66176660/146998105-1cdfe724-6078-49eb-bfd8-c72082c4090f.png)

**Comment by postman-staff** [comment]
@vstrelstov Glad to know that helped :)

I believe that shouldn't affect the running of the app, but if you do observe any unusual activity of the app rather than the expected flow. Feel free to add them over to the request you've submitted and we will take it from there.

via @Raj-postman | Postman Support

**Comment by vstrelstov** [comment]
Unfortunately, it turned out to be a one-time solution, so the issue occurs again after reboot

**Comment by m3dbedb** [comment]
I have the same issue, under centos 7 postman since 9.x.x version never starts after postman updates. Message "prepare workspaces" hands until system reboot.

**Comment by postman-staff** [comment]
@vstrelstov I believe the disable GPU action was done during the launching of the app and not on the system variable level as shared in doc. Could you try the same and let me know if that still persists?

* * *
@m3dbedb Could you please submit a support request: https://support.postman.com/hc/en-us/requests/new to help us look into this further?

If you could share the application logs and system details as per [this article here](https://support.postman.com/hc/en-us/articles/360047765713-How-to-open-my-logs-directory), would be really helpful to check this from here.

via @Raj-postman | Postman Support

**Comment by postman-staff** [comment]
@vstrelstov Just checking to see if setting value for system variable helped?

@m3dbedb Let me know if the request was raised, ticket Id can help us check further.

via @Raj-postman | Postman Support

**Comment by postman-staff** [comment]
I'll be closing this next on our end as we haven't heard back. If the issue persists, please reach out to us by submitting a support request: https://www.postman.com/support ​so we can investigate further.

via @Raj-postman | Postman Support

**Comment by twitharshil** [comment]
Thanks for reporting!
Seems to be a genuine issue that is reproducible after applying the updates on the Linux platform. We have sent a fix in Postman v9.8.3 for this issue. Marking the ticket closed for now, feel free to re-open if the issue is still reproducible on Postman v9.8.3 onwards :)
